src/taxpasta/domain/model/observation_matrix.py

```diff
@@ -0,0 +1,35 @@
+# Copyright (c) 2022, Moritz E. Beber, Maxime Borry, Jianhong Ou, Sofia Stamouli.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+"""Provide a description of an observation matrix."""
+
+
+import pandas as pd
+import pandera as pa
+from pandera.typing import Series
+
+
+class ObservationMatrix(pa.SchemaModel):
```

Comment:
Even after our walthrough of yesterday, I had 2 min of blanking out wondering what was this ObservationMatrix. Maybe renaming to WideObservationTable or something similar would help (something consistent with the tidy alter ego).

Reply:
Yes, I will do that.

Reply:
It probably also doesn't help that this solution for a schema with an arbitrary number of columns is not intuitive.